packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
```

Comment:
Since the code in this PR starts to use a real Stripe API for CI, the customers will start to pile up in SUB_PLAT_DEV in Stripe.

I've added this script to delete old customers from functional tests. I haven't added any jobs in CircleCI to automatically run it, though. Would like to get some eyes on it before doing that, probably in a follow-up issue.

It's easy enough to run manually, though - e.g.:
```
SUBHUB_STRIPE_APIKEY=rk_test_SOMEKEYHERE \
  node packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js \
  --dry-run
```
By default, this should delete customers with email addresses matching `/signin.*@restmail.net/` and older than 1 day. There are a few options, namely `--dry-run` which only *lists* the deletion candidates rather than actually deleting.

And this definitely is not something that should *ever* be run with a production Stripe key - only a test key from SUB_PLAT_DEV